sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/util/IdentityUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.azure.identity.implementation.util;
+
+import com.azure.core.credential.TokenRequestContext;
+import com.azure.core.experimental.credential.TokenRequestContextExperimental;
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.identity.implementation.IdentityClientOptions;
+
+public class IdentityUtil {
```

Comment:
Make this `final`. Add javadoc and include a private ctor as it doesn't require instantiation.

Reply:
Add a private constructor to prevent instantiation